en.md

```diff
@@ -72,15 +72,9 @@ Microsoft/Bing is certified under the “Privacy Shield” US-EU data protection
 
 We provide a blog in which we publish various posts on topics about our activities in order to circulate our website and its contents on the internet and to increase our visitor numbers. 
 
-We use the “Tumblr” blogging platform to operate the blog. The legal basis for this processing of your personal data is Art. 6 Abs. 1 lit. f of the General Data Protection Regulation. Tumblr is operated by Tumblr Inc., 35 East 21st St, Ground Floor, New York, NY 10010, USA.
+We use the “Ghost” blogging platform to operate the blog. Ghost does not have access to any of your personal information. For information on data protection with this third-party supplier, go to: https://ghost.org/privacy/.
```

Comment:
Wasn't IP a personal information? They do have access to that obviously.

Reply:
@DoHe Good point. I updated it.